crates/gossip/src/respond/initiate.rs

```diff
@@ -98,6 +98,12 @@ impl K2Gossip {
 
         // Update the peer's max op data bytes to reflect the amount of data we're sending ids for.
         // The remaining limit will be used for the DHT diff as required.
+        tracing::debug!(
```

Comment:
Would quite like a human friendly display of byte sizes here but didn't think we wanted to introduce a library. Thoughts?

Reply:
[do we need a library?](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0ccd064d4ca9c73882de1150592049f2) : )